ingestion/src/metadata/orm_profiler/orm/types/hex_byte_string.py

```diff
@@ -51,6 +52,13 @@ def process_result_value(self, value: str, dialect) -> Optional[str]:
         if not value:
             return None
         self.validate(value)
+
+        bytes_value = bytes(value)
```

Comment:
@MilanBariya why do we convert this to a `bytes` object. Normally it should already be one as at this stage it would have gone through the `self.validate()`.

Are you able to share an example of how the data is displayed in the sample data?

Reply:
Sure @TeddyCr , Let me share example in slack.

Reply:
i’m converting that in `bytes`  becaue result was in `memoryview`

here is the before after result
<img width="1412" alt="Screenshot 2022-12-13 at 1 13 38 PM" src="https://user-images.githubusercontent.com/52292922/207272707-9e714633-f610-4f2b-9db1-dabd8fca63f5.png">

<img width="1419" alt="Screenshot 2022-12-13 at 1 13 58 PM" src="https://user-images.githubusercontent.com/52292922/207272679-9a674606-9ce9-4277-93aa-1be323eea407.png">